modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/services/GlobalL0Service.scala

```diff
@@ -37,20 +50,36 @@ object GlobalL0Service {
     l0GlobalSnapshotClient: L0GlobalSnapshotClient[F],
     globalL0ClusterStorage: L0ClusterStorage[F],
     lastGlobalSnapshotStorage: LastSnapshotStorage[F, GlobalIncrementalSnapshot, GlobalSnapshotInfo],
-    singlePullLimit: Option[PosLong]
+    singlePullLimit: Option[PosLong],
+    maybeMajorityPeerIdSet: Option[NonEmptySet[PeerId]] = none
```

Comment:
I don't think you need a default here as well. I think if we set defaults to frequently we loose control over which value is used for which setup. Compiler will not bother you if you forgot to set a value.

Reply:
@TheMMaciek Ok, but then we'll need to coordinate a release with snapshot-streams won't we? Otherwise it will break. (I wonder if it needs to be updated regardless.)

Reply:
changed